war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -0,0 +1,463 @@
+      DataDocument document = dataStorage.readDocumentIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), Collections.singletonList(LumeerConst.Organization.ATTR_ORG_DATA));
+      return document != null ? document.getDataDocument(LumeerConst.Organization.ATTR_ORG_DATA) : null;
+   }
+
+   /**
+    * Reads specified additional information about the given organization.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param infoDataAttribute
+    *       attribute of the organization additional info
+    * @return value of the organization info attribute
+    */
+   public String readOrganizationInfoData(final String organizationId, final String infoDataAttribute) {
+      DataDocument document = dataStorage.readDocumentIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), Collections.singletonList(LumeerConst.Organization.ATTR_ORG_DATA));
+      DataDocument infoDataDocument = document.getDataDocument(LumeerConst.Organization.ATTR_ORG_DATA);
+      return infoDataDocument != null ? infoDataDocument.getString(infoDataAttribute) : null;
+   }
+
+   /**
+    * Updates whole organization info document.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param infoData
+    *       info document including more attributes and values
+    */
+   public void updateOrganizationInfoData(final String organizationId, final DataDocument infoData) {
+      DataDocument infoDataDocument = new DataDocument(LumeerConst.Organization.ATTR_ORG_DATA, infoData);
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, infoDataDocument, organizationIdFilter(organizationId));
+   }
+
+   /**
+    * Updates specified attribute in the organization info document.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param infoAttribute
+    *       attribute of the organization additional info to update
+    * @param value
+    *       new value of the given attribute
+    */
+   public void updateOrganizationInfoData(final String organizationId, final String infoAttribute, final String value) {
+      updateOrganizationInfoData(organizationId, new DataDocument(infoAttribute, value));
+   }
+
+   /**
+    * Drops the given attribute in the organization info document.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param infoAttribute
+    *       attribute of the organization additional info to drop
+    */
+   public void dropOrganizationInfoDataAttribute(final String organizationId, final String infoAttribute) {
+      DataDocument infoDocument = readOrganizationInfoData(organizationId);
+      infoDocument.remove(infoAttribute);
+      DataDocument document = new DataDocument(LumeerConst.Organization.ATTR_ORG_DATA, infoDocument);
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, document, organizationIdFilter(organizationId));
```

Comment:
Are you sure it works? Update doesn't drop any attribute. You must must `dropAttribute` function.